ui/packages/catalog-ui-search/src/main/webapp/component/layer-item/layer-item.less

```diff
@@ -42,6 +42,21 @@
         top: 0px;
     }
 
+    .layer-interactions {
+        text-align: right;
+    }
+
+    .layer-interactions > * {
```

Comment:
✏️ Since we have a homogeneous group of elements, can we update this to `.layer-interactions > button`?